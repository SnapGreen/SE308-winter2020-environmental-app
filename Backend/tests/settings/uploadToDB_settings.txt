settings check:
	SETTINGS: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/scripts/settings.txt
	DATADIR: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/data/
	FDADIR: FDA/
	FDADATADIR: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/data/FDA/
	SUFFIX_LEN: 4
	PRODS_PER_JSON: 250
	FB_WRITES_PER_DAY: 16500
	TESTDIR: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/tests/
	TEST_POPDB_SETTINGS: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/tests/settings/populateDB_settings.txt
	TEST_UPLOADTODB_SETTINGS: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/tests/settings/uploadToDB_settings.txt
	SPLIT_PREFIX: branded_food_
	OUTFILE_END: .json
	UPLOAD_SLEEP: 1
	LOGDIR: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/logs/
	UPLOADLOGDIR: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/logs/uploads/
	LASTUPLOAD: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/data/FDA/branded_food_0532.json
	SERVER_POPULATED: false
	DONE_UPLOADING: false
	USAGE:
		Usage: ./uploadToDB.sh [OPTIONS] (-h for help)
	HELP:
		Usage: ./uploadToDB.sh [OPTIONS] (-h for help)
		OPTIONS:
			-b: upload without prompt
			-s: print settings only
			-h: print help
