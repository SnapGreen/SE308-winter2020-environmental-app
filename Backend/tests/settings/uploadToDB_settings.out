settings check:
	SETTINGS: settings.txt
	DATADIR: ../data/
	FDADIR: FDA/
	FDADATADIR: ../data/FDA/
	SUFFIX_LEN: 4
	PRODS_PER_JSON: 250
	FB_WRITES_PER_DAY: 16500
	SPLIT_PREFIX: branded_food_
	OUTFILE_END: .json
	UPLOAD_SLEEP: 1
	LOGDIR: ../logs/
	UPLOADLOGDIR: ../logs/uploads/
	LASTUPLOAD: ../data/FDA/branded_food_0110.json
	SERVER_POPULATED: false
	DONE_UPLOADING: false
	USAGE:
		Usage: ./uploadToDB.sh [OPTIONS] (-h for help)
	HELP:
		Usage: ./uploadToDB.sh [OPTIONS] (-h for help)
		OPTIONS:
			-b: upload without prompt
			-s: print settings only
			-h: print help
